cypress/integration/search-ui.spec.js

```diff
@@ -57,28 +51,23 @@ describe(`${playground} playground test`, () => {
   })
 
   it('click on facets', () => {
-    cy.get(HIT_ITEM_CLASS).eq(0).contains('Counter-Strike')
-
     const checkbox = `.ais-RefinementList-list .ais-RefinementList-checkbox`
     cy.get(checkbox).eq(1).click()
-
-    if (playground === 'react') cy.contains('1,939')
-    if (playground === 'angular') cy.contains('1939')
-
-    cy.contains('Counter-Strike').should('not.exist')
+    cy.wait(1000)
+    cy.get(HIT_ITEM_CLASS).eq(1).contains('Team Fortress Classic')
+    cy.get(HIT_ITEM_CLASS).eq(1).contains('4.99 $')
   })
 
   it('Search', () => {
-    cy.get('.ais-SearchBox-input').type('orwell')
+    cy.get('.ais-SearchBox-input').type('Half-Life')
```

Comment:
I don't think that you were testing search in lower case but the old test was in lower case so just in case.